guides/v2.3/graphql/queries.md

```diff
@@ -1,35 +1,164 @@
+
+* A set of products to be displayed. This can include the entire catalog or those that match customer-specified criteria.
+* Customer data. With a customer token, a query can retrieve basic information about a customer as well as billing and shipping addresses, wishlists, order history, and other sensitive data.
+* Shopping cart contents. GraphQL supports both guest and logged-in customer carts.
+* Store configuration values, including theme and CMS settings, the currency code, and supported countries.
+
+The Magento REST GET endpoints retrieve a wide variety of information on behalf of the merchant. Many of these endpoints are for retrieving backend information. For example, the `GET /V1/customers/search` endpoint can be used to find a subset of customers that meet certain criteria, such as those that live in a particular state or have a birthday this month.  Likewise, the `GET /V1/invoices` endpoint can return all the recently-generated invoices. This type of functionality is not required for the frontend, so it is not available in GraphQL queries. The queries are designed to improve the customer's user experience by quickly retrieving the data needed to render pages. 
+
+## Structure of a query
+
+A query contains the following elements:
+
+* The optional keyword `query`. If no keyword is specified at the beginning of a request, the processor assumes the request is a query.
+* An operation name for your local implementation. This name is required if you include variables. Otherwise, it is optional.
+* The query name
+* The terms to search for. The terms can be in the form of objects, attributes, or a combination. Queries that don't require search terms obtain their context from the customer's authorization token or store ID, both of which are specified in the header of the call. 
+* The output object, which specifies which data the query returns. 
+
+The following example shows the structure of the `cart` query:
+
+```text
+query myCartQuery{
+  cart(cart_id: String!): Cart
+}
+```
+
+In the preceding example, `myCartQuery` identifies your implementation of the `cart` query.  `cart_id` is a non-nullable string that defines the cart to query. (The exclamation point indicates the value is non-nullable.) The `Cart` output object defines which fields to return. 
+
+Now let's fully define a query:
+
+```text
+query myCartQuery{
+  cart(cart_id: "1WxKm8WUm3uFKXLlHXezew5WREfVRPAn") {
+    cart_id
+    items {
+      id
+      qty
+    }
+    billing_address {
+      firstname
+      lastname
+      postcode
+      }
+    shipping_addresses {
+      firstname
+      lastname
+      postcode
+    }
+  }
+}
+```
+
+In this example, we've supplied a cart ID as input, (which was generated by the `createEmptyCart` mutation). The output includes the `cart_id` as well as selected information about the items in the card and the billing and shipping addresses.
```

Comment:
maybe link `createEmptyCart` to https://devdocs.magento.com/guides/v2.3/graphql/reference/quote.html